.github/workflows/build-test-publish-images.yml

```diff
@@ -55,8 +55,6 @@ jobs:
           ci/lint-dockerfiles.sh
   compute-matrix:
     runs-on: ubuntu-latest
-    container:
-      image: rapidsai/ci-conda:latest
```

Comment:
This is just some string manipulation in bash + `jq` + `yq`... I don't think we need to pay the cost of pulling a container image, and I think we can pretty safely just accept whatever `jq` / `yq` version GitHub is shipping in its images (ref: https://github.com/actions/runner-images/tree/main/images/ubuntu).